gwpy/frequencyseries/frequencyseries.py

```diff
@@ -435,6 +435,11 @@ def to_pycbc(self, copy=True):
             epoch = None
         else:
             epoch = self.epoch.gps
+        if self.f0.to('Hz').value:
+            raise ValueError(
+                "Cannot convert FrequencySeries to PyCBC with f0 = {self.f0}. "
```

Comment:
The f-string prefix character is missing here:

```suggestion
                f"Cannot convert FrequencySeries to PyCBC with f0 = {self.f0}. "
```